src/containers/genePage/index.js

```diff
@@ -76,7 +71,22 @@ const GenePage = ({geneId}) => {
     return null;
   }
 
-  const genomeLocation = getSingleGenomeLocation(data.genomeLocations);
```

Comment:
I don't believe this change is intentional. It seems to have reverted the code to an earlier state.